charts/brigade/templates/gateway-generic-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{ if .Values.genericGateway.enabled }}
+{{ $fullname := include "brigade.genericGateway.fullname" . }}
```

Comment:
We need to define `"brigade.genericGateway.fullname"` in `_helpers.tpl`, perhaps like so:
```
{{- define "brigade.genericGateway.fullname" -}}
{{ include "brigade.fullname" . | printf "%s-generic-gateway" }}
{{- end -}}
```

Reply:
thanks, I was wondering where I should place that... May I ask what method did you use to find it (other than `helm install charts/brigade --dry-run`)?

Reply:
@dgkanatsios I actually installed to test, but yes, `--dry-run` will catch it _if_ this feature is enabled when running the command: 
```
 $ helm install -n testbrigade charts/brigade --set genericGateway.enabled=true --dry-run
Error: render error in "brigade/templates/gateway-generic-service.yaml": template: brigade/templates/gateway-generic-service.yaml:5:20: executing "brigade/templates/gateway-generic-service.yaml" at <{{template "brigade....>: template "brigade.genericGateway.fullname" not defined
```
We have a ticket here to just add a test target running `--dry-run` installs to catch these things -- but this is a tricky case of a certain flag is needed as well... maybe for ci we turn 'em all on?

Reply:
got it thanks. Had some difficulty in running --dry-run though, mentioned about Kashti missing (although it's of course there in the charts/ folder), will try again